internal/cli/roles.go

```diff
@@ -0,0 +1,303 @@
+package cli
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/auth0/auth0-cli/internal/ansi"
+	"github.com/auth0/auth0-cli/internal/prompt"
+	"github.com/spf13/cobra"
+	"gopkg.in/auth0.v5/management"
+)
+
+// errNoRoles signifies no roles exist in a tenant
+var errNoRoles = errors.New("there are currently no roles")
+
+var (
+	roleID = Argument{
+		Name: "Role ID",
+		Help: "Id of the role.",
+	}
+	roleName = Flag{
+		Name:       "Name",
+		LongForm:   "name",
+		ShortForm:  "n",
+		Help:       "Name of the role.",
+		IsRequired: true,
+	}
+	roleDescription = Flag{
+		Name:      "Description",
+		LongForm:  "description",
+		ShortForm: "d",
+		Help:      "Description of the role.",
+		// IsRequired: true,
+	}
+)
+
+func rolesCmd(cli *cli) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "roles",
+		Short:   "Manage resources for roles",
+		Long:    "Manage resources for roles.",
+		Aliases: []string{"role"},
+	}
+
+	cmd.SetUsageTemplate(resourceUsageTemplate())
+	cmd.AddCommand(listRolesCmd(cli))
+	cmd.AddCommand(showRoleCmd(cli))
+	cmd.AddCommand(createRoleCmd(cli))
+	cmd.AddCommand(updateRoleCmd(cli))
+	cmd.AddCommand(deleteRoleCmd(cli))
+
+	return cmd
+}
+
+func listRolesCmd(cli *cli) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "list",
+		Aliases: []string{"ls"},
+		Args:    cobra.NoArgs,
+		Short:   "List your roles",
+		Long: `List your existing roles. To create one try:
```

Comment:
Can you please run the file through `gofmt`?

Reply:
Yeah, I think this is special gofmt logic.